src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.ts

```diff
@@ -106,7 +108,32 @@ export class VehicleTechnicalRecordComponent implements OnInit, AfterViewInit {
     }
   }
 
-  isAnyFormInvalid(forms: Array<CustomFormGroup | CustomFormArray>) {
+  getVehicleDescription(techRecord: TechRecordModel, vehicleType: VehicleTypes | undefined): string {
+    switch (vehicleType) {
+      case VehicleTypes.TRL:
+        return techRecord.vehicleConfiguration ?? '';
+      case VehicleTypes.PSV:
+        return techRecord.bodyMake && techRecord.bodyModel ? `${techRecord.bodyMake}-${techRecord.bodyModel}` : '';
+      case VehicleTypes.HGV:
+        return techRecord.make && techRecord.model ? `${techRecord.make}-${techRecord.model}` : '';
+      default:
+        return 'Unknown Vehicle Type';
+    }
+  }
+
+  createTest(isComplete?: string): void {
+    if (isComplete) {
+      this.router.navigate(['test-records/create-test/type'], { relativeTo: this.route });
+    } else {
+      confirm(
```

Comment:
Would alert() be better? confirm() suggests they have an option to continue or cancel.

Reply:
Suggestion applied.